test/dynamo/test_functions.py

```diff
@@ -1835,6 +1835,24 @@ def fn(x):
         ref = opt_fn(x)
         self.assertEqual(ref, res)
 
+    def test_broadcast_in_set(self):
+        param = torch.zeros(5)
+        param2 = torch.zeros(5, 10)
+
+        tensor_list = set()
+        tensor_list.add(param2)
+        assert param not in tensor_list
+
+        def fn(param, param2):
+            tensor_list = set([param2])
+            return param in tensor_list
+
+        opt_fn = torch.compile(fn, fullgraph=True)
```

Comment:
No need to fully torchcompile. Better compile it with `CompileCounter` and assert that there's one frame traced. I believe that `fullgraph=True` succeeds when there are no frames traced.

Otherwise, at least add `backend="eager"`

Reply:
Hmm I think fullgraph=True is the same as nopython?